From 532117de297ecedb121f20fdc122ad83482e9b48 Mon Sep 17 00:00:00 2001
From: "Richard W.M. Jones" <rjones@redhat.com>
Date: Wed, 28 Aug 2013 18:45:14 +0100
Subject: [PATCH] javadoc: Install javadoc in $(datadir)/javadoc/libguestfs
 (without -java-$(version)).

Apparently this is the normal place for javadoc, and not
where we installed it before.
---
 java/Makefile.am | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/java/Makefile.am b/java/Makefile.am
index 7cb22e9..524302f 100644
--- a/java/Makefile.am
+++ b/java/Makefile.am
@@ -115,8 +115,8 @@ doc-stamp: $(java_sources)
 	touch $@
 
 install-data-hook:
-	mkdir -p $(DESTDIR)$(datadir)/javadoc/$(PACKAGE_NAME)-java-$(PACKAGE_VERSION)
-	cp -a api/* $(DESTDIR)$(datadir)/javadoc/$(PACKAGE_NAME)-java-$(PACKAGE_VERSION)
+	mkdir -p $(DESTDIR)$(datadir)/javadoc/$(PACKAGE_NAME)
+	cp -a api/* $(DESTDIR)$(datadir)/javadoc/$(PACKAGE_NAME)
 
 # Tests (not comprehensive).
 
-- 
1.8.3.1

